lib/metamodelutil.js

```diff
@@ -64,6 +65,14 @@ function createNameTable(priorModels, metaModel) {
         'Participant': concertoNs,
         'Transaction ': concertoNs,
         'Event': concertoNs,
+        'StringMapKeyType': concertoNs,
```

Comment:
Not sure I understand this. StringMapKeyType is not defined in the `concerto@1.0.0` namespace - that is a type from the metamodel. What are we trying to do here?

Reply:
The intention is that any type used within a map context should have its namespace resolved. 

I guess my question is - why differentiate between types on the metamodel vs those on the root model when resolving namespace - why should namespace apply to only root types, but not in the case of primitives? The metamodel is versioned, after all.

If I think about this from the point of view that a primitive type will _always_ be a primitive type it doesn't really make sense to version it to a namespace, and in that case there is no need to generate values namespace values for primitives in the context of a `resolve` command output.

Reply:
Feels like I may not be seeing the forest for the trees.., I think it would be useful if you can share a scenario under which the resolve function is necessary, from a client perspective.

Reply:
What this code is doing (before you changes) is saying that anywhere we see the unqualified type `Event` being used (et al) it should be resolved as `concerto@1.0.0.Event`.

The `concerto` namespace is special in that it is managed by the `ModelManager` implicitly. I.e. any model file can import `Event` without having to define it. Here is the code that adds the `concerto` namespace to the model manager on construction.
https://github.com/accordproject/concerto/blob/main/packages/concerto-core/lib/basemodelmanager.js#L126

Your code change is saying that `StringMapKeyType` is *also* defined in the `concerto` namespace, which is not correct.